runsc/boot/loader.go

```diff
@@ -627,7 +635,24 @@ func (l *Loader) startContainer(spec *specs.Spec, conf *Config, cid string, file
 		caps,
 		l.k.RootUserNamespace())
 
-	procArgs, err := newProcess(cid, spec, creds, l.k)
+	var pidns *kernel.PIDNamespace
```

Comment:
nit: You can assign pidns here and change in case a PID namespace is in the Spec:
pidns := l.k.RootPIDNamespace()
if ns, ok := specutils.GetNS(specs.PIDNamespace, spec); ok {
   ...
}

Reply:
Thanks for your suggestion !

I need pidns in line 648.
If I assign pidns in the beginning, I can't determine when to assign a new pid namespace to pidns.